From 28dbef159ada2784bb6001d003d9d609be6b2889 Mon Sep 17 00:00:00 2001
From: Ganapatrao Kulkarni <gkulkarni@caviumnetworks.com>
Date: Fri, 10 Jun 2016 12:52:45 +0530
Subject: [PATCH 0233/1200] booti, arm64: align kernel start address to 2MB.

linux kernel start address for arm64 should be aligned
to 2MB(refer Documentation/arm64/booting.txt)

Adding code to make sure kernel loaded at 2MB aligned dram address.
This fixes the kernel boot issue with u-boot, if it starts at
2MB unaligned address.

Signed-off-by: Ganapatrao Kulkarni <gkulkarni@caviumnetworks.com>
---
 cmd/bootm.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmd/bootm.c b/cmd/bootm.c
index df0bbe19ca..995fef429a 100644
--- a/cmd/bootm.c
+++ b/cmd/bootm.c
@@ -19,6 +19,7 @@
 #include <asm/byteorder.h>
 #include <linux/ctype.h>
 #include <linux/err.h>
+#include <linux/sizes.h>
 #include <u-boot/zlib.h>
 
 DECLARE_GLOBAL_DATA_PTR;
-- 
2.22.0

